### PR TITLE
keep jump/file history when using :split

### PR DESCRIPTION
**Pull request by robinvd**
i was missing the beheviour from vim

### CODE REVIEW

File: helix-term/src/commands.rs [hunk 1]
```diff
@@ -3992,12 +3992,23 @@ fn split(cx: &mut Context, action: Action) {
...
+    let gutters = view.gutters.clone();
 
     cx.editor.switch(id, action);
 
     // match the selection in the previous view
     let (view, doc) = current!(cx.editor);
     view.offset = offset;
+    view.jumps = jumps;
+    view.docs_access_history = docs_access_history;
+    view.last_modified_docs = last_modified_docs;
```
Comment (pickfire) [inline helix-term/src/commands.rs:4008]: Why do we need to change these?

File: helix-term/src/commands.rs [hunk 2]
```diff
@@ -3992,12 +3992,23 @@ fn split(cx: &mut Context, action: Action) {
...
+    view.object_selections = object_selections;
+    view.gutters = gutters;
```
Comment (archseer) [inline helix-term/src/commands.rs:4010]: How about doing this inside `switch`, under the split case? You should be able to just `view.clone()` the current view

**Review by archseer — CHANGES_REQUESTED** [review]

File: helix-term/src/commands.rs (hunk 2 above)
Comment (robinvd) [reply helix-term/src/commands.rs:4010]: i thought that as well, but the offset was already here, so i just included it haha

Comment (robinvd) [reply helix-term/src/commands.rs]: should be fixed!

**Review by archseer — APPROVED** [review]
Thanks!

**Pull request merged**